docs/source/conf.py

```diff
@@ -65,9 +65,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '1.0.6'
+version = '1.1.0'
 # The full version, including alpha/beta/rc tags.
-release = '1.0.6'
+release = '1.1.0'
```

Comment:
I think that will break readthedocs since it uses this string for version info but isn't molecule aware.